NewForce-Cohort-9/tabloid-full-stack-api-all-stars
# Ticket #23 View All User Profiles

### Description

The functionality for an admin to view all users has been added.

Given an admin has logged in to the app, when they are on the Category list page, they should be able to select `User Profiles` on the header and see a list of all users listed alphabetically by user display name. 

[Trello Ticket #23](https://trello.com/c/9ef3dsN6/23-23-view-all-user-profiles)

### Type of Change


### Testing Instructions

1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout <il-viewallusers>` 
5. If you don't see the most recent version of my files, `git pull origin <il-viewallusers>`
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
9. Click the execute with debugger solid green button in purple Visual Studio to launch the api
10. In your terminal, launch the program within the root folder with the command `npm start`
11. In browser, please go to `Login` on the nav bar and log in with this email: `foo@bar.com`
12. This should redirect you back to the home page. 
13. You should  see `User Profiles` on the nav bar.
14. Select `User Profiles` to be directed to `/users` where you should see a list of all users ordered alphabetically by display name and include their full names and user type.
15. Select the `Create Category` button and enter `Test` in the category name input and click `Save`
16. Logout
17. Login with this email `user@example.com`
18.  You should NOT see `User Profiles` on the nav bar as this user is not an admin.



### Checklist:
